madgraph_gridpack.py

```diff
@@ -17,6 +17,25 @@ def prepare_default_card(self):
         if glob.glob(f'{cards_path}/*_cuts.f'):
             os.system(f'cp {cards_path}/*_cuts.f {job_files_path}')
 
+
+    def get_proc_card(self):
```

Comment:
this doesn't do anything in `madgraph_gridpack.py`. It's only for `main.py:get_proc_card` web api purposes